localstripe/resources.py

```diff
@@ -678,6 +678,11 @@ def _api_delete(cls, id):
         schedule_webhook(Event('customer.deleted', obj))
         return super()._api_delete(id)
 
+    @classmethod
+    def _api_retrieve_sources(cls, id):        
+        obj = cls._api_retrieve(id) # return 404 if does not exist
```

Comment:
Does this code really retrieve a list of sources that belong to a customer?

Reply:
I believe so because the sources are set here:
https://github.com/adrienverge/localstripe/blob/f5a2e0bc4330624741c4e3e655648d5c00f49452/localstripe/resources.py#L629

And then updated here:
https://github.com/adrienverge/localstripe/blob/f5a2e0bc4330624741c4e3e655648d5c00f49452/localstripe/resources.py#L739
and: 
https://github.com/adrienverge/localstripe/blob/f5a2e0bc4330624741c4e3e655648d5c00f49452/localstripe/resources.py#L754

So I'm just returning that value from the customer object